conjur/wrapper/keystore_adapter.py

```diff
@@ -75,8 +75,14 @@ def is_keyring_accessible(cls):
         try:
             # Send a dummy request to test if the keyring is accessible
             keyring.get_password('test-system', 'test-accessibility')
-        except keyring.errors.KeyringError as keyring_error:
-            logging.debug(keyring_error)
+        except keyring.errors.KeyringError:
             return False
 
         return True
+
+    @classmethod
+    def configure_keyring_log(cls):
+        """
+        Method to configure the keyring logs
+        """
+        logging.getLogger('keyring').setLevel(logging.WARNING)
```

Comment:
very nice.
I would also add the logLevel to the function accepted parameters or rename the function to configure_keyring_log_to_warning. (better to add it to the function parameter I think.)

Reply:
done

Reply:
fixed to .INFO instead of warning